bindings/java/nz/mega/sdk/DelegateMegaGlobalListener.java

```diff
@@ -2,6 +2,25 @@
 
 import java.util.ArrayList;
 
+/**
```

Comment:
This comment block goes to the *head* of the Java file, and it's usually also not in JavaDoc (it's informational in the code only, not to be extracted by JavaDoc).

The way it's done now is, that it's the documentation of the `DelegateMegaGlobalListener` class, which it's not, but rather some meta-data for the file itself.